modules/rostests/apitests/gdi32/GetObject.c

```diff
@@ -701,6 +722,25 @@ Test_Region(void)
     DeleteObject(hRgn);
 }
 
+void Test_CursorIcon()
+{
+    BITMAP bmp;
+    HBITMAP hbmMask = CreateBitmap(32, 64, 1, 1, (VOID*)src_mask);
+    /*
+         This test is valid until Win2003Srv sp2.
+         On XP sp3 GetObject reports a 32x32 bitmap.
+    */
+    GetObjectW(hbmMask, sizeof(bmp), &bmp);
+    ok(bmp.bmWidth == (bmp.bmHeight / 2), "ERR UNICODE CursorIcon RECT got %ldx%ld\n", bmp.bmWidth, bmp.bmHeight);
+    ok(bmp.bmHeight == 64, "ERR UNICODE CursorIcon Height got %ld\n", bmp.bmHeight);
+    DeleteObject(hbmMask);
+ 
+    GetObjectA(hbmMask, sizeof(bmp), &bmp);
+    ok(bmp.bmWidth == (bmp.bmHeight / 2), "ERR ANSI CursorIcon RECT got %ldx%ld\n", bmp.bmWidth, bmp.bmHeight);
+    ok(bmp.bmHeight == 64, "ERR ANSI CursorIcon Height got %ld\n", bmp.bmHeight);
+    DeleteObject(hbmMask);
```

Comment:
Why there's an ANSI version of this...?

Reply:
You ask for UNICODE version. I just know de UNICODE version that is GetObjectW() and ANSY version that is GetObjectA(). Can you be more specific about your first comment?

Reply:
You have had a mix of Wide and ANSI calls before I have written the first commit and my comment was that you should only use the Unicode variants of the API calls you use. This is pretty self-explanatory, isn't it?